test/test_trixi.jl

```diff
@@ -153,7 +153,14 @@ macro test_nowarn_mod(expr, additional_ignore_content = String[])
                                      r"┌ Warning: Keyword argument letter not supported with Plots.+\n└ @ Plots.+\n",
                                      r"┌ Warning: `parse\(::Type, ::Coloarant\)` is deprecated.+\n│.+\n│.+\n└ @ Plots.+\n",
                                      # TODO: Silence warning introduced by Flux v0.13.13. Should be properly fixed.
-                                     r"┌ Warning: Layer with Float32 parameters got Float64 input.+\n│.+\n│.+\n│.+\n└ @ Flux.+\n"]
+                                     r"┌ Warning: Layer with Float32 parameters got Float64 input.+\n│.+\n│.+\n│.+\n└ @ Flux.+\n",
+                                     "WARNING: Method definition pretty_form_utf(Base.Val{:energy_potential}) in module TrixiTestModule at /home/runner/work/Trixi.jl/Trixi.jl/examples/paper_self_gravitating_gas_dynamics/elixir_eulergravity_jeans_instability.jl:121 overwritten in module TrixiTestModule on the same line (check for duplicate calls to `include`).\n",
```

Comment:
That's a bit extreme. Could you please reduce all these lines using some regexes?